onebusaway-android/src/main/java/org/onebusaway/android/util/RegionUtils.java

```diff
@@ -114,6 +115,31 @@ public static ObaRegion getClosestRegion(ArrayList<ObaRegion> regions, Location
     }
 
     /**
+     * Get the region name if it is available. If there is a custom url instead of a region from
+     * the region api, then hash the custom url and return it.
+     *
+     * @return regionName
+     */
+    public static String getObaRegionName() {
+        String regionName = null;
+        ObaRegion region = Application.get().getCurrentRegion();
+        if (region != null && region.getName() != null) {
+            regionName = region.getName();
+        } else if (Application.get().getCustomApiUrl() != null) {
+            MessageDigest digest;
```

Comment:
Could you please move the URL hash code to it's own method?  I think this will make it easier to read here and understand what is happening.